rts/lra/coordinator/pom.xml

```diff
@@ -133,6 +164,19 @@
               <groupId>org.sonatype.plugins</groupId>
               <artifactId>nexus-staging-maven-plugin</artifactId>
             </plugin>
+
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
similar to the comment with the `application.host` property. I think this is not used.

Reply:
okay